warehouse/templates/manage/release.html

```diff
@@ -73,7 +73,7 @@ <h2 class="heading-wsubtitle__heading">Release Version {{ release.version }}</h2
                 <i class="fa fa-hashtag" aria-hidden="true"></i>
                 View Hashes
               </a>
-              {% set title = "Delete File From" %}
+              {% set title = "Delete File" %}
```

Comment:
This makes the title of the modal say "Delete File {project name}":

<img width="397" alt="screen shot 2018-04-02 at 3 35 19 pm" src="https://user-images.githubusercontent.com/294415/38214903-a598c83e-368b-11e8-8c58-cd0bf11f258b.png">

I think we need to find a way to have differing title and button texts.

Reply:
To be explicit here: We want the title to be `Delete file from {project name}` and the button to be `Delete file`